Jenkinsfile

```diff
@@ -243,6 +243,32 @@ pipeline {
     }
 
     stages {
+        stage('check_for_changed_files') {
+          steps {
+            script {
+	      def all_files = new ArrayList()
+
+              def changeLogSets = currentBuild.changeSets
+              for (int i = 0; i < changeLogSets.size(); i++) {
```

Comment:
Does `for (changeSet  in currentBuild.changeSets)` not work? (I don't actually know)

Reply:
I think it might... if I could figure out how to get a changeSet :(